apps/website/components/alert/README.md

```diff
@@ -229,12 +229,6 @@ This class must be applied with `.alert` to render an app-level alert.
 
 <doc-demo src="/demos/alert/multiple-app-level-alerts-ng.html" />
 
-### Summary of Options
-
-<DocAlertSummaryOfOptions table="text-styles" />
-
-Deprecation: Since v4, we will no longer handle setting `aria-live` and announcing the message for you by default. Based on the application use case, you can use the new `ClrAriaLiveService` to make announcements when they make sense for a user to hear about updates or loading status changes. This will result in of removing few inputs provided by the component such as `clrPolite`, `clrAssertive`, `clrOff`
```

Comment:
Lets move this to the api.md tab and put it into a cds-alert. 

core components are a little funny in markdown but I'venot had time to wrap them into a vue component. Here is syntax that worked in other docs:

```html
  <cds-alert-group type="default" status="warning">
  <cds-alert closable>Acknowledge
  <cds-alert-actions>
  <cds-button>Action Link</cds-button>
  </cds-alert-actions>
  </cds-alert>
  </cds-alert-group>
```